### PR TITLE
use drone for all flavours

### PR DESCRIPTION
**Pull request by tobiasKaminsky**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -2,18 +2,23 @@ pipeline:
   test:
     image: nextcloudci/android:android-15
     commands:
+      # uncomment gplay
+      - sed -i '/com.google.*.gms/s/^.*\/\///g' build.gradle
```
Comment (mario) [inline .drone.yml]: Afaik this is not the only thing that needs uncommenting? There's also a dependency?

File: .drone.yml [hunk 2]
```diff
@@ -22,3 +27,9 @@ pipeline:
       - ANDROID_TARGET=android-24
       - ANDROID_ABI=armeabi-v7a
       - LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu/:/opt/android-sdk-linux/tools/lib64/gles_mesa/
+
+  matrix:
+    FLAVOUR:
+      - Generic
+      - Gplay
+      - Modified
```
Comment (mario) [inline .drone.yml]: New line please.

Comment (tobiasKaminsky) [inline .drone.yml]: New line? At the end? But..why?

File: .drone.yml (hunk 1 above)
Comment (tobiasKaminsky) [inline .drone.yml]: This uncomments
compile 'com.google.android.gms:play-services:10.2.4'
apply plugin: 'com.google.gms.google-services'

Comment (mario) [inline .drone.yml]: Ah ok.

File: .drone.yml (hunk 2 above)
Comment (mario) [inline .drone.yml]: Because you always need a new line at the end of a file? :D

**Comment by mario** [comment]
Build failed looks like.

**Pull request closed without merging**